torch/jit/frontend.py

```diff
@@ -439,6 +439,12 @@ def build_Num(ctx, expr):
         r = ctx.make_range(expr.lineno, expr.col_offset, expr.col_offset + len(value))
         return Const(r, value)
 
+    @staticmethod
+    def build_Str(ctx, expr):
+        value = str(expr.s)
+        r = ctx.make_range(expr.lineno, expr.col_offset, expr.col_offset + len(value))
```

Comment:
nit: you don't account for quotation marks in `len` here

Reply:
at this point the quotations have been removed

Reply:
Yeah, that's my point. Does `col_offset` point to the column where the quotation mark is, or to the first column of the actual string?

Reply:
We should get back to this too.

Reply:
Unfortunately expr doesn't provide any information the length of the string including quotation marks. So there's no way to distinguish "a" vs """a""". I think the best thing to do here is just point to the first quotation beginning the string.